docs/Training-on-Microsoft-Azure.md

```diff
@@ -31,7 +31,9 @@ view the documentation for doing so [here](#custom-instances).
 1. [Move](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/copy-files-to-linux-vm-using-scp)
    the `ml-agents` sub-folder of this ml-agents repo to the remote Azure
    instance, and set it as the working directory.
-2. Install the required packages with `pip3 install .`.
+2. Install the required packages with `pip3 install ...`.
```

Comment:
i think the ... is a typo.  also, if you are doing pip3 install mlagents, you dont need this line